ncm-metaconfig/src/main/pan/components/metaconfig/config.pan

```diff
@@ -9,6 +9,8 @@ include 'components/${project.artifactId}/schema';
 
 bind '/software/components/${project.artifactId}' = ${project.artifactId}_component;
 
+'version' ?= '${no-snapshot-version}';
```

Comment:
An habit! I almost never use `=`. For me `?=` tends to be the default assignment operator. And this is defined like this in all configuration modules. In this particular case, I agree that it doesn't make sense to allow somebody to define another value and it may make sense to change for `= `. A topic for next workshop?!